spring-cloud-contract-wiremock/src/main/java/org/springframework/cloud/contract/wiremock/WireMockConfiguration.java

```diff
@@ -97,9 +97,6 @@ private void registerStubs() throws IOException {
 						this.resourceLoader);
 				String pattern = stubs;
 				if (!pattern.contains("*")) {
-					if (!pattern.endsWith("/")) {
```

Comment:
To tell you the truth I don't really remember why we've added this in the first place. Nor can I break the tests! Do you have some samples that you took that blew up this piece of code? I'm pretty sure you've pasted them somewhere